### PR TITLE
API Remove GraphQL

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-admin/issues/1799

### CODE REVIEW

File: src/Forms/FormRequestHandler.php [hunk 1]
```diff
@@ -77,7 +77,6 @@ public function __construct(Form $form)
     public function Link($action = null)
     {
         // Forms without parent controller have no link;
-        // E.g. Submission handled via graphql
```
Comment (GuySartorelli) [inline src/Forms/FormRequestHandler.php:80]: That's still valid - graphql is still supported and submissions handled via graphql from the front-end will presumably meet this criterion

File: src/ORM/Connect/NullDatabase.php [hunk 2]
```diff
@@ -8,8 +8,7 @@
 /**
  * Utility class required due to bad coupling in framework.
  * Not every framework execution should require a working database connection.
- * For example, when generating class and config manifests for deployment bundles,
- * or when generating code in a silverstripe/graphql schema build.
+ * For example, when generating class and config manifests for deployment bundles
```
Comment (GuySartorelli) [inline src/ORM/Connect/NullDatabase.php]: Still valid - this presumably applies to a front-end schema still.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/Forms/FormRequestHandler.php (hunk 1 above)
Comment (emteknetnz) [reply src/Forms/FormRequestHandler.php:80]: I don't think it was ever valid - graphql was not used to submit forms

File: src/ORM/Connect/NullDatabase.php (hunk 2 above)
Comment (emteknetnz) [reply src/ORM/Connect/NullDatabase.php]: Reverted

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when other PRs are happy

**Review by GuySartorelli — APPROVED** [review]
Looks like aside from the rebase, the only change since last approval was the explicit addition of `schemaDataType` on `GridField` which seems like a sensible decision to me.

**Review by GuySartorelli — APPROVED** [review]
re-approved most recent changes

**Pull request merged**